core/graphelier-service/api/hndlrs/playbackhandler.go

```diff
@@ -80,7 +80,7 @@ func StreamPlayback(env *Env, w http.ResponseWriter, r *http.Request) error {
 	case mErr == nil && tErr != nil:
 		loader = &CountIntervalLoader{Instrument: instrument, Datastore: env.Datastore, Count: rateMessages}
 	case tErr == nil && mErr != nil:
-		loader = &TimeIntervalLoader{Instrument: instrument, Datastore: env.Datastore, Interval: uint64(delay / rateRealtime)}
+		loader = &TimeIntervalLoader{Instrument: instrument, Datastore: env.Datastore, Interval: uint64(delay * rateRealtime)}
```

Comment:
seems to be a pretty big change to the logic, what's happening here?

Reply:
Completely agree, rateRealtime is supposed to be a ratio corresponding to how much faster/slower we should be playing back compared to real time. Say the request specifies `rateRealtime=0.5`, then we should be playing back at 50% real time speed. To determine the interval at which to load more messages, we should multiply with the delay instead of dividing. For example with a delay of 4s and a rateRealtime of 0.75, we should be sending 3 seconds of data every 4 seconds.